pypeit/core/telluric.py

```diff
@@ -1167,195 +1191,189 @@ def mask_star_lines(wave_star, mask_width=10.0):
 
     return mask_star
 
-def sensfunc_telluric(wave, counts, counts_ivar, counts_mask, exptime, airmass, std_dict, telgridfile,
-                      ech_orders=None,
-                      polyorder=8, mask_abs_lines=True,
+def sensfunc_telluric(wave, counts, counts_ivar, counts_mask, exptime, airmass, std_dict,
+                      telgridfile, ech_orders=None, polyorder=8, mask_abs_lines=True,
                       delta_coeff_bounds=(-20.0, 20.0), minmax_coeff_bounds=(-5.0, 5.0),
-                      sn_clip=30.0, ballsize=5e-4, only_orders=None, maxiter=3, lower=3.0, upper=3.0, tol=1e-3, popsize=30, recombination=0.7,
-                      polish=True, disp=False,
+                      sn_clip=30.0, ballsize=5e-4, only_orders=None, maxiter=3, lower=3.0,
+                      upper=3.0, tol=1e-3, popsize=30, recombination=0.7, polish=True, disp=False,
                       debug_init=False, debug=False):
     """
-    Function to compute a sensitivity function by simultaneously fitting the sensfunc and a telluric model to a standard
-    star spectrum.  The sensitivity function is defined to be S_lam = F_lam/(counts/s/A) where F_lam is in units of
-    1e-17 erg/s/cm^2/A, and so the sensitivity function has units of (erg/s/cm^2/A)/(counts/s/A) = erg/cm^2/counts
+    Provided a standard star spectrum, compute a best-fit telluric model.
```

Comment:
"Provided a standard star spectrum, compute a best-fit telluric model" 

This line does not describe what this code does at all and I'm not sure why it was added. Is there something about the original detailed documentation that is not clear?

Reply:
Thanks for the new text.  Changed.